alibi_detect/cd/base.py

```diff
@@ -740,6 +873,46 @@ def predict(self, x: Union[np.ndarray, list], return_p_val: bool = True, return_
             cd['data']['distance_threshold'] = distance_threshold
         return cd
 
+    @abstractmethod
+    def get_config(self) -> dict:
+        """
+        Get the detector's configuration dictionary.
+
+        Returns
+        -------
+        The detector's configuration dictionary.
+        """
+        cfg: Dict[str, Any] = {
+            'version': __version__,
+            'config_spec': __config_spec__,
+            'name': 'LSDDDrift'
```

Comment:
For this and some others, what is the reason we're not using `self.__class__.__name__` here? I note that we are using it for e.g. `BaseUnivariateDrift`

Reply:
This is done for the detectors with tensorflow/pytorch backends, as we want the name to be `LSDDDrift` instead of `LSDDDriftTF` etc. (We want the non-backend specific wrapper names as this is what we'll use to re-instantiate at load time)